PSKoans/Koans/Cmdlets 2/AboutOutCmdlets.Koans.ps1

```diff
@@ -115,23 +115,20 @@ Context 'Out-* Cmdlets' {
             Out-String is a fantastic and simple way to turn complex objects into strings that look just
             like PowerShell's default console formatting.
         #>
-
+    
         It 'creates string representations of data' {
-            $HomeFolder = Get-Item -Path $HOME
-
-            $HomeFolder | Should -Not -BeNullOrEmpty
-            $String = $HomeFolder | Out-String
-
-            # Fill in this here-string with the data you'd see in your console from calling Get-Item $HOME
+            #Create a hashtable that pipes into Out-String
+            $String = '__' | Out-String
+    
             @"
-
-
-    Directory: __
-
-
-Mode                LastWriteTime         Length Name
-----                -------------         ------ ----
-______         __________________                ____
+    
+    Name                           Value                                                                                                                                                                                         
+    ----                           -----                                                                                                                                                                                         
+    ColumnA                        Value1                                                                                                                                                                                        
+    ColumnB                        Value2                                                                                                                                                                                        
```

Comment:
Given we're using a hashtable here (_fantastic_ idea btw, tons simpler than my original idea on both sides of the fence), we should probably swap the `Column` names for `Key` names to better reflect what's expected here. 🙂

Reply:
Works for me. Updated.